Magick++/lib/Magick++/Include.h

```diff
@@ -517,7 +519,7 @@ namespace Magick
   using MagickCore::LZMACompression;
   using MagickCore::LZWCompression;
   using MagickCore::PizCompression;
-  using MagickCore::Pxr24Compression;
+  using MagickCore::Pxr24Compression;  
```

Comment:
Could you please remove the extra space?

Reply:
Oups, I'll clean that for sure

For the ReadEXRImage, I'll check that too

Reply:
What is weird is that convert is completly capable of reading Dwaa exr without the lines I'm adding

Reply:
(The extra space have been fixed in an amend commit)

Reply:
What that part of the code does it setting the compression on the image. It does not need it to be able to read the file. You can see that with when you use identify:

```
I:\exr>identify -format "Compression: %[compression]\n" *.exr
Compression: Zip
Compression: None
Compression: ZipS
Compression: ZipS
Compression: Piz
Compression: Piz
Compression: Piz
Compression: Piz
Compression: Piz
```